### PR TITLE
111 i18n button and dropdown menu

### PR DESCRIPTION
**Pull request by Uprising5034**

### CODE REVIEW

**Comment by Hayor4real** [comment]
nice great job

File: src/components/credentials/credentials.css [hunk 1]
```diff
@@ -30,6 +30,12 @@
   align-items: center;
 }
 
+.credentials-bottom {
+  display: flex;
```
[the remark this answers is not in the document]
Comment (Luca-Terrazzan) [reply src/components/credentials/credentials.css:34]: @jules

[the remark this answers is not in the document]
Comment (Luca-Terrazzan) [reply src/components/credentials/credentials.css:34]: `@jules`

File: src/components/languageSelect/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import { useEffect, useState } from "react"
+
+import i18n from "../../i18n"
+import Menu from "../menu"
+
+import "./style.css"
+
+const LanguageSelect = () => {
+  const [selectedLanguage, setSelectedLanguage] = useState("en")
```
Comment (Auenc) [inline src/components/languageSelect/index.js:9]: Could we use a const for the default language. We could also try and load a desired default language via the environment variables

**Review by Auenc — CHANGES_REQUESTED** [review]

**Review by Auenc — APPROVED** [review]

**Pull request merged**